### PR TITLE
Translatable template apps

### PR DESCRIPTION
**Pull request by kryskool**
Do not merge yet

### CODE REVIEW

File: tcms/templates/case/clone.html [hunk 1]
```diff
@@ -20,53 +21,53 @@
 		<input id="value_plan_id" type="hidden" name="plan_id" value="{{ test_plan.plan_id }}" />
 	</div>
 
-	<h2>Clone TestCase(s)</h2>
+	<h2>{% trans "Clone TestCase(s)" %}</h2>
 	<div class="Detailform border-1">
 		<div class="editor" cellspacing="0" cellspan="0">
 			<div class="listinfo">
-				<label class="strong title">Select Plan </label>
+				<label class="strong title">{% trans "Select Plan " %}</label>
```
Comment (atodorov) [inline tcms/templates/case/clone.html]: You don't need the trailing space in source string.

File: tcms/templates/case/edit.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {% extends "tcms_base.html" %}
+{% load i18n %}
 {% load static %}
 {% block subtitle %}
     {% if test_case %}
-        Edit test case - {{ test_case.summary }}
+        {% trans "Edit test case" %} - {{ test_case.summary }}
     {% else %}
-        Add new test case
+        {% trans "Add new test case" %}
```
Comment (atodorov) [inline tcms/templates/case/edit.html]: FYI this file will be removed as part of #730 which will also be marked for translations so you probably want to exclude these changes and save yourself from resolving a conflict later.

File: tcms/templates/case/get_details.html [hunk 3]
```diff
@@ -1,14 +1,15 @@
+{% load i18n %}
 {% load comments %}
 {% load extra_filters %}
 <td id="id_case_text_{{ forloop.counter }}" colspan="12">
 	<table class="case_detail">
 		<tr>
 			<td valign="top" colspan='2'>
-				<h4>Text:</h4>
+                            <h4>{% trans "Text:" %}</h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: Trailing `:` in string. Elsewhere we've excluded these from the source strings.

File: tcms/templates/case/clone.html (hunk 1 above)
Comment (kryskool) [reply tcms/templates/case/clone.html]: Ok

File: tcms/templates/case/edit.html (hunk 2 above)
Comment (kryskool) [reply tcms/templates/case/edit.html]: ok

File: tcms/templates/case/get_details.html (hunk 3 above)
Comment (kryskool) [reply tcms/templates/case/get_details.html]: ok

**Comment by kryskool** [comment]
@atodorov thanks for the review

**Comment by atodorov** [comment]
@kryskool - do you mind reverting the changes for `case/edit.html`, fixing the minor issues from review and rebasing to the latest master? I would like to merge this as-is and include it in the next release which is planned for next week.

If you would like to translate the new source strings just let me know how much time you think you will need and I will do my best to include the translations in the new release.

**Comment by kryskool** [comment]
hi @atodorov 

I can change it this week end, thanks for the tips

Regards

File: tcms/templates/case/edit.html (hunk 2 above)
Comment (kryskool) [reply tcms/templates/case/edit.html]: Hi @atodorov 
I've rebase from master, #730 is merge but [case/edit.html](https://github.com/kiwitcms/Kiwi/blob/master/tcms/templates/case/edit.html) have not been removed

**Comment by kryskool** [comment]
@atodorov it's done can you check ?

File: tcms/templates/case/get_details.html [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
 	<table class="case_detail">
 		<tr>
 			<td valign="top" colspan='2'>
-				<h4>Text:</h4>
+                            <h4>{% trans "Text" %}:</h4>
 				<div class="content">{{ test_case.text|markdown2html }}</div>
 			</td>
 			<td valign="top" rowspan="2" width="200">
-				<h4><span class="title strong">Attachment:</span></h4>
+                            <h4><span class="title strong">{% trans "Attachment:" %}</span></h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: :

File: tcms/templates/case/get_details.html [hunk 5]
```diff
@@ -17,50 +18,50 @@ <h4><span class="title strong">Attachment:</span></h4>
 						</a>
 					</li>
 					{% empty %}
-					<li class="grey">No attachment found</li>
+                                        <li class="grey">{% trans "No attachment found" %}</li>
 					{% endfor %}
 				</ul>
-				<h4><span class="title strong">Component:</span></h4>
+                                <h4><span class="title strong">{% trans "Component:" %}</span></h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: :

File: tcms/templates/case/get_details.html [hunk 6]
```diff
@@ -17,50 +18,50 @@ <h4><span class="title strong">Attachment:</span></h4>
...
 				<ul class="ul-no-format">
 					{% for component in components %}
 					<li id="display_component" >{{ component.name }}</li>
 					{% empty %}
-					<li class="grey">No component found</li>
+                                        <li class="grey">{% trans "No component found" %}</li>
 					{% endfor %}
 				</ul>
-				<h4><span class="title strong">Tag:</span></h4>
+                                <h4><span class="title strong">{% trans "Tag:" %}</span></h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: :

File: tcms/templates/case/get_details.html [hunk 7]
```diff
@@ -17,50 +18,50 @@ <h4><span class="title strong">Attachment:</span></h4>
...
 				<ul class="ul-no-format">
 					{% for tag in tags %}
 					<li>{{ tag }}</li>
 					{% empty %}
-					<li class="grey">No tag found</li>
+                                        <li class="grey">{% trans "No tag found" %}</li>
 					{% endfor %}
 				</ul>
-				<h4><span class="title strong">bug:</span></h4>
+                                <h4><span class="title strong">{% trans "bug:" %}</span></h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: :

File: tcms/templates/case/get_details.html [hunk 8]
```diff
@@ -17,50 +18,50 @@ <h4><span class="title strong">Attachment:</span></h4>
...
 				<ul class="ul-no-format">
 					{% for bug in test_case.get_bugs %}
 					<li>
 						<a href="{{ bug.get_full_url }}">{{ bug }}</a>
 						{% if bug.case_run_id %}
-						<span class="grey"> - From Run</span>
+                                                <span class="grey">{% trans " - From Run" %}</span>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: `-`

File: tcms/templates/case/get_details.html [hunk 9]
```diff
@@ -17,50 +18,50 @@ <h4><span class="title strong">Attachment:</span></h4>
...
-					<li class="grey">No bug found</li>
+                                        <li class="grey">{% trans "No bug found" %}</li>
 					{% endfor %}
 				</ul>
 			</td>
 		</tr>
 		<tr>
 			<td colspan='3'>
-				<h4>Notes:</h4>
+                            <h4>{% trans "Notes:" %}</h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: :

File: tcms/templates/case/get_details.html [hunk 10]
```diff
@@ -17,50 +18,50 @@ <h4><span class="title strong">Attachment:</span></h4>
...
 				<div class="content">{{ test_case.notes|urlize|linebreaksbr }}</div>
 			</td>
 		</tr>
 		<tr>
 			<td colspan="3" style="border:none; background:none">
-				<h4>Comments:</h4>
+                            <h4>{% trans "Comments:" %}</h4>
```
Comment (atodorov) [inline tcms/templates/case/get_details.html]: :

File: tcms/templates/case/get_details.html (hunk 4 above)
Comment (kryskool) [reply tcms/templates/case/get_details.html]: good point

File: tcms/templates/case/edit.html [hunk 11]
```diff
@@ -107,41 +108,41 @@
...
 				<div class="editor-side">
-					<h4 class="strong">Setup</h4>
+                                    <h4 class="strong">{% trans "Setup" %}</h4>
 					<div class="form" id="form_text">{{ form.text }}</div>
 				</div>
 			</fieldset>
                     {% if notify_form %}
 			<fieldset class="no-border notification">
-				<label class="strong">Send new case instance after editing <label class="grey">(The author of this case will receive the notification by default)</label>:</label>
+                            <label class="strong">{% trans "Send new case instance after editing " %}<label class="grey">{% trans "(The author of this case will receive the notification by default)" %}</label>:</label>
```
Comment (atodorov) [inline tcms/templates/case/edit.html]: trailing space

File: tcms/templates/plan/preview.html [hunk 12]
```diff
@@ -48,5 +49,5 @@
 <hr />
 {% endif %}
 {% empty %}
-{"msg":"The plan you specified does not exist in the database."}
+{"msg":"{% trans "The plan you specified does not exist in the database." %}"}
```
Comment (atodorov) [inline tcms/templates/plan/preview.html]: I don't think this is a valid construct in Django templates. Just looks like printing JSON inside HTML.

File: tcms/templates/registration/login.html [hunk 13]
```diff
@@ -28,7 +28,7 @@
         </div>
         <div class="form-group">
           <div class="col-xs-8 col-sm-offset-2 col-sm-6 col-md-offset-2 col-md-6">
-            <span class="help-block">Forgot <a href="{% url "tcms-password_reset" %}" tabindex="6">password</a>?</span>
+              <span class="help-block">{% trans "Forgot" %} <a href="{% url "tcms-password_reset" %}" tabindex="6">{% trans "password" %}</a>?</span>
```
Comment (atodorov) [inline tcms/templates/registration/login.html]: These should be a single string surrounded by `blocktrans`. Same for the rest of the registration templates.

**Pull request merged**